spring-xd-dirt/src/main/java/org/springframework/xd/dirt/server/admin/AdminServerApplication.java

```diff
@@ -189,4 +194,18 @@ public Filter httpPutFormContentFilter() {
 	public BatchDatabaseInitializer batchDatabaseInitializer() {
 		return new XdBatchDatabaseInitializer();
 	}
+
+
+	@Autowired
+	private ScheduledAnnotationBeanPostProcessor postProcessor;
+
+	@Autowired
+	@Qualifier("framework")
+	private TaskScheduler frameworkTaskScheduler;
+
+	@PostConstruct
+	public void forceScheduledTaskExecutor() {
+		postProcessor.setScheduler(frameworkTaskScheduler);
```

Comment:
Interesting technique :smile: 

A couple of thoughts...

* Perhaps we should open a JIRA against SF to fall back to some well-known bean name when there's more than one scheduler in the context?
* We should probably document someplace how users can override this "framework" scheduler if they want multiple threads in the `TPTS`